tests/integration_test.py

```diff
@@ -37,6 +38,9 @@ def pulsevideo_cmdline(source_pipeline=None):
             '--bus-name-suffix=test']
 
 
+TestCtx = namedtuple('TestCtx', 'bus pulsevideod')
```

Comment:
I'd prefer to have a separate fixture to get a bus rather than returning this test context object, but the dependencies between these two are complex and I don't know how to express it within the constraints of py.test's DI system.